src/nemos/basis.py

```diff
@@ -1327,16 +1330,16 @@ def split_by_feature(
         Suppose the basis has **m components** (features), each with $b_i$ basis functions and $n_i$ inputs.
         The total number of features, `N`, is:
 
-        \[
+        $$
         N = b_1 \cdot n_1 + b_2 \cdot n_2 + \ldots + b_m \cdot n_m
-        \]
+        $$
 
         Given an input array of shape `(..., N, ...)`, the method slices it into `m` sub-arrays,
         each of shape:
 
-        \[
+        $$
         (..., n_i, b_i, ...)
-        \]
+        $$
 
         Here:
         - $n_i$ is the number of inputs processed by the **i-th basis component**.
```

Comment:
```suggestion

        - $n_i$ is the number of inputs processed by the **i-th basis component**.
```
Need an empty line here to render the docs correctly